packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/load-app.md

```diff
@@ -0,0 +1,68 @@
+### Set up the Okta configuration settings
+
+Use the required [configuration settings](#okta-org-app-integration-configuration-settings) to initialize your Sign-In Widget and your Auth JS instance:
+
+* `clientId`: Your client ID &mdash; `${yourClientId}`
+* ` issuer`: The authorization server in your Okta org (for example, `https://${yourOktaDomain}/oauth2/default`)
+* `useInteractionCodeFlow`: Set this option to `true` to enable the [Interaction Code flow](/docs/concepts/interaction-code/#the-interaction-code-flow) in the embedded Widget.
```

Comment:
What happens if you don't enable this flag? Is there an error we might want to mention on this page?

Reply:
If you don't enable this flag, you are not using the Interaction flow in the Widget. I don't think there would be an error, but I'll try it and see what happens. Thanks